brokerbot.go

```diff
@@ -32,22 +32,24 @@ var (
 	ctx context.Context
 
 	finnhubClient *finnhub.DefaultApiService
-	geminiClient *http.Client
-
-	timeSinceLastHeartbeat time.Time
+	geminiClient  *http.Client
 )
 
 type tickerType int
 
 const (
 	crypto tickerType = iota
 	stock
+
+	botHandle = "@BrokerBot"
+	botPrefix = "!stonks"
```

Comment:
Why'd we bring back the !stonks prefix?

Reply:
I preferred it, slightly less characters to type. Now both work.